Code of conduct

joshbruce opened the pull request:
**Marked version:** 0.3.17

We're still setting up the community things related to bringing Marked back to life. 

See GitHub recommendations on open source projects, specifically the [CODE_OF_CONDUCT](https://opensource.guide/code-of-conduct/)

As a [contributor](https://github.com/markedjs/marked/blob/master/AUTHORS.md), what else do you think should be included here? (Use review functionality?? And comments.)

styfle commented on CODE_OF_CONDUCT.md:
```diff
@@ -0,0 +1,16 @@
+## Practices
+
+- **Final doesn't exist:** It is assumed that each of us is working to the best of their ability to make Marked better using the knowledge and skills we each have at the time.
+- **Perfection is the unattainable goal:** The long we wait to make something perfect, the longer our users have to wait for the features and functionality the want and need to be delighted with Marked. Having said that, we will always strive to continuously improve in all areass.
```
The `long` or the `longer` we wait?

styfle replied:
`areass` should probably be `areas`
Maybe run this through a spell check first.

joshbruce replied:
lol - Fair. Was mainly throwing it up here to see what y'all wanted to add; or, if y'all thought these were not worth having. Updated title to be more explicit of where the PR is, sorry about that.

styfle replied:
I think it would be easiest to grab an existing code of conduct and remove anything we don't agree with. That's what GitHub recommends:

> Wherever you can, use prior art. The Contributor Covenant is a drop-in code of conduct that is used by over 40,000 open source projects, including Kubernetes, Rails, and Swift.
> The Django Code of Conduct and the Citizen Code of Conduct are also two good code of conduct examples.

joshbruce replied:
I saw that recommendation as well - and think it may be a good one. 

Under normal circumstances, I would facilitate a gathering where we actually talked about this stuff - good team building exercise - it just so happens to wind up with cultural norms at the end. 

Thought this might be a good alternative and give us a chance to actually build something together as a team, which I suppose removing things might cover as well.

Which one do you think is a better for us to start with??

joshbruce replied:
Which of the two mentioned in the quote...?? 

(I also tend toward the 8fold one, which is why I picked the format, I prefer it over legalize or dry one, you know? Don't want it to be milquetoast, of course, I guess that's really up to the contributors in the end.)

Anyway...probably not focused enough on it to really take in all the potential impacts before feeling comfortable taking action.

https://8fold.pro/about

styfle replied:
The 8fold about page doesn't look like a code of conduct.
How about the node.js [CODE_OF_CONDUCT.md](https://github.com/nodejs/admin/blob/master/CODE_OF_CONDUCT.md)

After all, this is a project that runs on node.

joshbruce replied:
Yeah, the 8fold one emphasizes the concept of culture all around as opposed to "thou shalt do this and not do that". But, it could be odd for people accustomed to the other type. I do appreciate the NodeJS one and don't have anything that I would fundamentally disagree with.

intcreator commented:
I agree with the idea of modifying an existing code of conduct.  We can save time that way and get started on the long list of issues.

joshbruce commented:
Updated using the NodeJS base. Added section at top: Our North Star. Just a recommendation.

@styfle: Passing final decision making authority to you, brother.

styfle commented on CODE_OF_CONDUCT.md:
```diff
@@ -1,16 +1,92 @@
+This Code of Conduct is adapted from the [Contributor Covenant][homepage],
+version 1.4, available at [http://contributor-covenant.org/version/1/4][version]
+
+## Appendix A: The report@nodejs.org Alias
```
Probably remove this appendix since its specific to node

joshbruce replied:
Agreed. Do you want to merge and takeover; or, I think you can work directly in the PR by hitting the "open this in GitHub Desktop"...?? Or, start a new PR and close?? I should probably catch up on the day job before they notice I'm missing...luckily they know this work is helping them too. ;)

styfle commented on CODE_OF_CONDUCT.md:
```diff
@@ -1,16 +1,92 @@
+review the [Moderation Policy] and [Requesting Moderation].
+
+[Moderation Policy]: https://github.com/nodejs/admin/blob/master/Moderation-Policy.md#moderation-policy
+[Requesting Moderation]: https://github.com/nodejs/admin/blob/master/Moderation-Policy.md#requesting-moderation
```
Should these documents be created?

joshbruce replied:
Hmmmm...didn't notice those. If the links are there - probably - having said that, I don't know if we're that far along and I'm thinking the changes to the AUTHORS page might help?? Think they can be removed for now, to be honest.

styfle replied:
Let's remove them

styfle commented:
@UziTech @davisjam Any thoughts on this?
